File: server/php/src/Presentation/Mappers/HostedTokenizationMapper.php
<?php

namespace OnlinePayments\ExampleApp\Presentation\Mappers;

use OnlinePayments\ExampleApp\Application\DTOs\HostedTokenization\ResponseDto;
use OnlinePayments\ExampleApp\Presentation\Models\HostedTokenization\Response;

class HostedTokenizationMapper
{
    public static function mapResponse(?ResponseDto $response): Response
    {
        return new Response(
            $response->hostedTokenizationId,
            $response->hostedTokenizationUrl
        );
    }
}

File: server/python/app/presentation/mappers/hosted_tokenization_mapper.py
from app.application.dtos.hosted_tokenization.response_dto import HostedTokenizationResponseDto
from app.presentation.models.hosted_tokenization.response import HostedTokenizationResponse

class HostedTokenizationPresentationMapper:

    @staticmethod
    def from_dto(response_dto: HostedTokenizationResponseDto) -> HostedTokenizationResponse:
        return HostedTokenizationResponse(
            hosted_tokenization_id=response_dto.hosted_tokenization_id,
            hosted_tokenization_url=response_dto.hosted_tokenization_url
        )